content/en/blog/2020/large-scale-security-policy-performance-tests/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Large Scale Security Policy Perfomance Tests
+subtitle: How does having security policies effect latency of requests
+description: How does having security policies effect latency of request.
+publishdate: 2020-09-02
+attribution: "Michael Eizaguirre (Google)"
+keywords: [test,security policy]
+---
+
+## Overview
+
+Istio has a wide range of Security policies which can be easily configured into systems of services. As the number of Security policies being applied increases it is important to understand how much latency these Security policies are adding to the system.
+
+This blog post goes over common Security policies use cases and how the number of Security policies or the number of specific rules in a Security policy can effect the overall latency of requests.
```

Comment:
Can we mention here is the very first step of performance test that focuses the latency, more could be done to measure other metrics (e.g. cpu, mem)

Reply:
Great updated it